css/metabox-320.css

```diff
@@ -1,7 +1,7 @@
 /* css for snippet */
 #snippet_preview {
 	position: relative;
-	width: 640px;
+	width: 552px;
```

Comment:
My guess is this should be 640px: see https://github.com/Yoast/wordpress-seo/pull/4729